src/pytorch_lightning/loggers/wandb.py

```diff
@@ -580,6 +581,10 @@ def _scan_and_log_checkpoints(self, checkpoint_callback: Checkpoint) -> None:
 
         # log iteratively all new checkpoints
         for t, p, s, tag in checkpoints:
+            if isinstance(p, torch.Tensor):
+                p = p.item()
+            if isinstance(t, torch.Tensor):
+                t = t.item()
```

Comment:
These code paths will never be taken. p is a path and t is a timestamp. Could you verify that #15543 fixes the issue for you? Thanks!